middleware-jackson/src/main/java/com/nytimes/android/external/store/middleware/jackson/JacksonStringParser.java

```diff
@@ -29,10 +30,10 @@ public JacksonStringParser(@Nonnull ObjectMapper objectMapper, @Nonnull Type typ
     }
 
     @Override
-    @Nullable
-    public Parsed call(@Nonnull String source) {
+    @SuppressWarnings({"PMD.EmptyCatchBlock"})
```

Comment:
My understanding of Rxjava2 is that nulls are not allowed anywhere. If source coming into parser is non null I feel that parser should be nonNull return as well.